version-matrix.yaml

```diff
@@ -0,0 +1,11 @@
+images:
+  - pytorch-ngc:
+    ngc_version: 23.12-py3
+    deepspeed_version: 0.13.0
+  - pytorch-ngc-hpc:
+    ngc_version: 23.12-py3
+    deepspeed_version: 0.13.0
```

Comment:
I think users would prefer to see the python version and pytorch version in this matrix as well.

I know they _could_ look it up from the ngc version but if I were a user I'd find that hella annoying.

Reply:
Added framework versions and python versions.